### PR TITLE
command to destroy a migration

### PR DESCRIPTION
**Pull request by josedonizetti**
Hey, I'm still learning Clojure, so any feedback is welcome! :)

### CODE REVIEW

File: src/migratus/database.clj [hunk 1]
```diff
@@ -232,6 +236,13 @@
           migration-down-name (str migration-name ".down.sql")]
       (.createNewFile (File. migration-dir migration-up-name))
       (.createNewFile (File. migration-dir migration-down-name))))
+  (destroy [this name]
+      (let [migration-dir (find-migration-dir (:migration-dir config))
+            migration-name (camel-snake-kebab/->kebab-case name)
+            pattern (re-pattern (str "[\\d]*-" migration-name ".*.sql"))
+            migrations (file-seq migration-dir)]
+            (if-let [files (filter #(re-find pattern (.getName %)) migrations)]
```
Comment (yogthos) [inline src/migratus/database.clj]: minor nitpick: better to use when-let if there's no negative case

File: test/migratus/test/core.clj [hunk 2]
```diff
@@ -69,3 +69,23 @@
       (down config 2)
       (is (empty? @ups))
       (is (empty? @downs)))))
+
+(defn- migration-exists? [name]
+    (let [migrations (file-seq (migratus.database/find-migration-dir "migrations"))
+          names (map #(.getName %) migrations)]
+          (seq (filter #(.contains % name) names))))
```
Comment (yogthos) [inline test/migratus/test/core.clj]: is there a reason to wrap the filter result with a seq?

**Comment by yogthos** [comment]
Looks good to me, I added a couple of minor comments otherwise it's good to merge I think. :)

**Comment by josedonizetti** [comment]
Suggestions applied, thank you! :)

**Comment by josedonizetti** [comment]
Should I change migratus-lein to support the destroy command?

**Comment by yogthos** [comment]
looks great, and yeah it would make sense to update the plugin as well

**Pull request merged**